get-substrate.sh

```diff
@@ -15,7 +15,7 @@ if [[ "$OSTYPE" == "linux-gnu" ]]; then
 		echo "Please refer to https://github.com/paritytech/substrate for setup information."
 	elif [ -f /etc/arch-release ]; then
 		echo "Arch Linux detected."
-		$MAKE_ME_ROOT pacman -Sy cmake gcc openssl-1.0 pkgconf git clang
+		$MAKE_ME_ROOT pacman -Sy --noconfirm cmake gcc openssl-1.0 pkgconf git clang
```

Comment:
```suggestion
		$MAKE_ME_ROOT pacman -Syu --noconfirm cmake gcc openssl-1.0 pkgconf git clang
```

Arch [does not support partial upgrades](//wiki.archlinux.org/index.php/System_maintenance#Partial_upgrades_are_unsupported), so `pacman -Syu` **must** be run before installing any packages.